Login

mKolibri opened the pull request:
Login registartion, not ended

mKolibri committed:
Added login and register pages.

mKolibri committed:
login, Registration

mkhitar-h reviewed (changes requested):
public	
src	
.gitignore	
README.md	
package-lock.json	
package.json
Above folders/files should be under the root of project (not in the login folder)
2. Where is Read.me wich is explained how to intall and run this APP.
3. Please clean/remove any file which you cannot explain why there are in the project (e.g: .React Simple App .gitingore, styles, README.me, logo, etc),

mKolibri committed:
Added working version,
now working login, ragistration, logout,
getUserInfo, getuserTAbles.
applied PR notes.
